app/util.py

```diff
@@ -6,7 +6,7 @@
 def get_model():
     global __loaded_model
     if not __loaded_model:
-        __loaded_model = pickle.load(open("/automl/ml/model.pkl", "rb"))
+        __loaded_model = pickle.load(open("/automl/ml/model.pt", "rb"))
```

Comment:
use the torch package to load the model, atm the container fails while unpickling